cli/src/commands/runTests.js

```diff
@@ -593,7 +593,8 @@ export async function run(argv: Argv): Promise<number> {
   const onlyChanged = Boolean(argv.onlyChanged);
 
   const cwd = process.cwd();
-  const cwdDefsNPMPath = path.join(cwd, "definitions", "npm");
+  const basePath = argv.path ? String(argv.path) : cwd;
+  const cwdDefsNPMPath = path.join(basePath, "definitions", "npm");
```

Comment:
Is path a new argument you just created for this purpose?

Reply:
Yep I added it only to be able to adjust the libdef path for tests.

Reply:
Not much, add an entry for it in the wiki, and add a `setup` method to `runTests`, so that the  new parameter appears when running `flow-typed run-tests --help`.

You'll need to change the `run` method to accept the new `Args` instead of `Argv`:

```javascript
import typeof Yargs from 'yargs';

export type Args = {
  _: Array<string>,
  path?: string,
};

export function setup(yargs: Yargs) {
  return yargs.usage(`$0 ${name} - ${description}`).options({
    path: {
      alias: 'p',
      describe: 'Some description...',
      type: 'string',
    }
  })
})

export async function run(args: Args) { ...
```

Reply:
Thanks! Added the setup function and wiki entry.